utils/frame/benchmarking-cli/src/pallet/writer.rs

```diff
@@ -315,12 +316,12 @@ fn get_benchmark_data(
 	let mut base_calculated_proof_size = 0;
 	// Sum up the proof sizes per component
 	for (_, slope, base) in proof_size_per_components.iter() {
-		base_calculated_proof_size += base;
+		base_calculated_proof_size = base_calculated_proof_size.max(*base);
 		for component in slope.iter() {
 			let mut found = false;
 			for used_component in used_calculated_proof_size.iter_mut() {
 				if used_component.name == component.name {
-					used_component.slope += component.slope;
+					used_component.slope = used_component.slope.max(component.slope);
```

Comment:
so we just take the steepest slope and the largest base among all keys (which could be from two different keys)

Reply:
Yea but by what do you want to prioritize? If we go by largest base then it possibly underestimates the slope and vice versa.